app/js/DocManager.js

```diff
@@ -150,6 +150,10 @@ module.exports = DocManager = {
     )
   },
 
+  getAllDeletedDocs(project_id, filter, callback) {
+    MongoManager.getProjectsDeletedDocs(project_id, filter, callback)
```

Comment:
Any reason not to call `MongoManager` directly outside here, given that there's no extra logic in this method?

Reply:
Separation of concerns: the http layer does not need to know how to actually action a request, but just how to transform the request inputs and produce a response from the output.